framework/transformer/bash/command.py

```diff
@@ -24,12 +25,15 @@ def export(_input: Command, _output: Bash) -> Bash:
     :param _output: the output bash script, converted from the command
     :return: the bash script that will be exported if required.
     """
+    content_to_add = f'export PATH="$PATH:{_input.export_folder}"'
+    file_contains_content = file_contains_value(exported_paths_path, content_to_add)
+    
```

Comment:
the code looks great, but it's not optimized and following design conventions. there are two issues:
- it performs a file read every time a module bash script is generated
- it performs an I/O operation at the domain/generation layer, where it's not expected to happen. all I/O calls should be in the installer layer

Reply:
Just implemented another solution. Please check again ☺️

Reply:
thank you! approved 😄